sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -770,6 +770,10 @@ def get_blob_client(
             blob_name = blob.name
```

Comment:
I wonder if `get_blob_client` should actually take in a `version_id` parameter like it does for Snapshot. I think it probably should. We would keep this parsing from the properties as well but the parameter would override what's in the props. Probably add this as an optional keyword after a `*,` since this method does not take a `kwargs` currently (and we should keep it that way). Don't forget to document it :).

This would apply for `ContainerClient.get_blob_client` as well (and asyncs)